go.mod

```diff
@@ -10,4 +10,4 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 )
 
-go 1.13
+go 1.14
```

Comment:
Supporting Go 1.13 for Geneva release

Reply:
@tsconn23 Thank you for the review, i have reverted to golang:1.13